src/com/unbounce/dogstatsd/core.clj

```diff
@@ -2,10 +2,11 @@
   (:import [com.timgroup.statsd StatsDClient NonBlockingStatsDClient ServiceCheck NoOpStatsDClient]))
 
 ;; In case setup! is not called, this prevents nullpointer exceptions i.e. Unit tests
-(defonce ^:private ^StatsDClient client
-  (NoOpStatsDClient.))
+(defonce ^:private ^StatsDClient client (NoOpStatsDClient.))
 
-(defn str-array [tags]
+(def ^:private ^"[Ljava.lang.String;" -empty-array (into-array String []))
```

Comment:
I was trying to look up the ^"[Ljava.lang.String;" thing and didn't know what to google for 😕

Reply:
It's a typehint for a string array